sympy/printing/pycode.py

```diff
@@ -0,0 +1,35 @@
+from sympy.printing.codeprinter import CodePrinter
+
+_kw_py2and3 = {
+    'and', 'as', 'assert', 'break', 'class', 'continue', 'def', 'del', 'elif',
+    'else', 'except', 'finally', 'for', 'from', 'global', 'if', 'import', 'in',
+    'is', 'lambda', 'not', 'or', 'pass', 'raise', 'return', 'try', 'while',
+    'with', 'yield'
+}
+_kw_only_py2 = {'exec', 'print'}
+_kw_only_py3 = {'False', 'None', 'nonlocal', 'True'}
```

Comment:
`None` is sort of a keyword in Python 2, since you cannot assign to it. I'm assuming that is how `reserved_words` is used.